packages/manager/apps/dedicated/client/app/ip/components/list/firewall/ip-ip-firewall.controller.js

```diff
@@ -120,7 +120,7 @@ export default /* @ngInject */ function IpFirewallCtrl(
           const newResult = result;
           const map = {
             key: name,
-            name: parseInt(name.replace('_', ''), 10),
+            name: key,
```

Comment:
Why key is not key and name isn't name. It isn't logical.

Reply:
I am going to use { key, name } instead it will work and be better for code